definitions/tools/merge_somatic_vcfs.cwl

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "merge several somatic VCFs into a multi-sample VCF"
+baseCommand: ["/bin/bash", "merge_somatic_vcfs.sh"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: 'zlskidmore/samtools:1.10'
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'merge_somatic_vcfs.sh'
+        entry: |
+            set -eou pipefail
+
+            vcfs="$1" #csv list of vcfs
+            samples="$2" #csv list of tumor sample names
+            normal_sample_name="$3"
+            
+            IFS=',' read -ra vcf_array <<< "$vcfs"
+            IFS=',' read -ra sample_array <<< "$samples"
+            
+            if [ "${#vcf_array[@]}" != "${#sample_array[@]}" ]; then
+                echo "ERROR: the number of vcfs must be equal to the number of sample names provided";
+                exit 1
+            fi
+            
+            # prep all the tumor vcfs for merging
+            for i in ${!vcf_array[*]}; do
+                echo "loop: $i";
```

Comment:
In our setup it may be more useful for debugging messages and the like to end up in stderr instead of stdout.

Reply:
removed